webui/src/lib/components/repositoryCreateForm.jsx

```diff
@@ -68,8 +63,90 @@ export const RepositoryCreateForm = ({ config, onSubmit, onCancel, error = null,
         }
     }, []);
 
+    const sampleCheckbox = (
+      <Form.Group controlId="sampleData" className="mt-3">
+          <Form.Check ref={sampleDataCheckbox} type="checkbox" label="Add sample data, hooks, and configuration" />
+      </Form.Group>
+    );
+
+    const flattenForm = !defaultNamespacePrefix;
+
+    const basicSettings = (
+      <div>
```

Comment:
Not sure you need the extra div wrapper. If you need it just so that you can export a single wrapper component, you can use `<>....</>` (short hand for `<React.Fragment>...</React.Fragment>`) instead.
[ref](https://react.dev/reference/react/Fragment)